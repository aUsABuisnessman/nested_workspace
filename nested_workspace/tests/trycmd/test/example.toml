args = ["test"]

[bin]
name = "cargo"

[fs]
cwd = "../../../../example"
