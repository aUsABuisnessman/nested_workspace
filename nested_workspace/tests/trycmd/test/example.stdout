
running 1 test

running 1 test
test tests::add_works ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in [..]s


running 1 test
test tests::sub_works ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in [..]s


running 1 test
test tests::div_works ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in [..]s


running 1 test
test tests::mul_works ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in [..]s

test nested_workspace ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in [..]s

